### DISCUSSION TITLE
Sveltekit example

### DISCUSSION THREAD

**Discussion by ctwhome** [cawa-93/vite-electron-builder#811] [Forks]
Did anyone try to make it run with sveltekit?

**Comment by AarushX** [comment]
https://github.com/HermanLederer/vite-electron-builder-svelte may work for you. It's not sveltekit, but I found a few results when I looked for sveltekit online.

**Reply by ctwhome** [reply]
I've tried quickly to convert that to sveltekit but without success yet

**Comment by van100j** [comment]
Created two forks, one for each — [SvelteKit](https://github.com/van100j/sveltekit-vite-electron-builder) and [Svelte](https://github.com/van100j/svelte-vite-electron-builder). 

Depending on the use case, `Svelte` might be good enough. We could use `SvelteKit` in Electron as a [static site generator (SSG), with `adapter-static` adapter](https://github.com/sveltejs/kit/tree/master/packages/adapter-static). Would consider `SvelteKit` to take advantage of its routing — might be useful when in need for multiple `BrowserWindow`s in Electron.
